src/Channel.cpp

```diff
@@ -340,3 +340,19 @@ void Channel::makeOperAnUser(std::string nickname) {
     else {}
         //throw ChannelException(USER_NOT_FOUND_ERR);
 }
+
+/**
+ * This function aims to find an operator user by the nickname.
+ * 
+ * @param nickname The nickname of the user.
+ * 
+ * @return `true` if the user is an operator, `false` otherwise.
+ */
+bool Channel::findOperUser(std::string nickname) const
```

Comment:
Yo lo llamaría isOper, isOperUser, ... algo así, más que find

Reply:
Mas que hacer un for buscando en `_operators`, existe ya `findOper` que devuelve un iterador al objeto que cumple la condición del nickname. Si es `_operators.end()`, entonces es `false`

Antes de implementar algo nuevo, se tiene que intentar reutilizar lo que hay ya para no hacer trabajo de más y empeorar la calidad del código.